php/php.editor/test/unit/data/testfiles/completion/lib/php56/classConst.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
```

Comment:
The rat does not check the license header of test files. However, we should add it to new test files if possible. Of course, we need not add it to *.completion, *.pass, and so on.

Reply:
I added license to both new test files.